scripts/update_teams.py

```diff
@@ -100,7 +105,7 @@ def create_team(org, name, description, repos):
     team._requester.requestJsonAndCheck("PUT", url, input={"permission": "push"})
 
     current_members = team.get_members()
-    member_handles = set([member.login for member in current_members])
+    member_handles = set([member.login.lower() for member in current_members])
```

Comment:
I can imagine that this will break. I'm not sure where github is case sensitive, but if the `register-github` case is any indication, there are places where github wants the exact match.